server/public/styles.css

```diff
@@ -330,6 +331,60 @@ body {
   background-color: lightgray;
 }
 
+.edit-page {
+  /* display: flex; */
+
+  /* justify-content: center; */
+
+  /* padding-top: 1rem; */
```

Comment:
😱

Reply:
Oh nooo 🙀 I was in such a rush to submit the PR to you before the end of the day that I forgot to double-check my CSS for the comments. Sorry! 😅